modules/Bio.js

```diff
@@ -0,0 +1,106 @@
+import Gallery from './Gallery.js'
+import Series from './Series.js'
+import Video from './Video.js'
+import Video2 from './Video2.js'
+import Gallery2 from './Gallery2.js'
+
+const Bio = () =>
```

Comment:
Looks like you missed the indentation here. See if  you can fix this.

Reply:
i have fixed it